### PR TITLE
Add test for end date handler

### PR DESCRIPTION
**Pull request by cwahbong**
RRRRRRRRRRRRRRRRRR

### CODE REVIEW

**Comment by qcl** [comment]
I'll try to fix CI setting and after that I'll reopen this PR so that this PR can be rebuild again via travis CI.

**Comment by qcl** [comment]
I'd tried to fix PR related settings.

File: .travis.yml [hunk 1]
```diff
@@ -2,8 +2,7 @@ language: python
 python:
 - "2.7"
 # install dependency
-# install: "pip install -r requirements.txt"
-install: "echo 'nothing to install for now.'"
+install: pip install -r requirements.txt
 # run tests
 # script: nosetests
```
Comment (qcl) [inline .travis.yml]: You wrote test script but not modify .travis.yml, so there is no test be run, please fix it , thanks.

File: .travis.yml [hunk 2]
```diff
@@ -2,12 +2,10 @@ language: python
 python:
 - "2.7"
 # install dependency
-# install: "pip install -r requirements.txt"
-install: "echo 'nothing to install for now.'"
-# run tests
-# script: nosetests
+install: pip install -r requirements.txt
 script: "echo 'nothing to test for now.'"
```
Comment (qcl) [inline .travis.yml]: Please remove this line because we have test now (once this PR be merged), thanks.

**Comment by cwahbong** [comment]
I've done some cleanups.

**Comment by qcl** [comment]
LGTM, :shipit:

**Pull request merged**